plugins/kiali-backend/src/index.ts

```diff
@@ -1,3 +1,2 @@
 export * from './service/router';
 export * from './plugin';
-export * from './dynamic/index';
```

Comment:
There should be the following line (coming from [here](https://github.com/janus-idp/backstage-plugins/pull/2284/files#diff-5dc70c80b770e0fe0a290423fdb0018b26795c0224daeb7d3f2b6366f895aeeaL17)) added instead of the removed line:

```
export { kialiPlugin as default } from './plugin'
```

Without this default export the plugin would not be found by the new backend system affect.